RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# How to have a release
+
+For Falter-releases, we usally do follow this workflow:
+
+## Have a pre-release
+
+At first, we trigger the buildbot to build a pre-release. For this, you need to adjust the `freifunk_release` file in the falter-common package. Adjust the string in `FREIFUNK_RELEASE` to something like `1.2.5-rc1` or similar. In addition, you might need to adjust `FREIFUNK_OPENWRT_BASE` to a proper OpenWrt-release like `21.02.1`. Once you've pushed the commit, the buildbot should start building the release automatically (-> webhook).
+
+As testing every configuration can be quite time-consuming, we usally involve the community into that process. Just write a short mail to the mailing list asking for some testing. :)
+
+## Do the actual release
+
+When everything is ready, do thing in this order:
```

Comment:
Grammar error in "do thing in this order".  Could be changed to "do the following"

Reply:
done in new version.